build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
   id 'jacoco'
   id 'io.spring.dependency-management' version '1.0.5.RELEASE'
   id 'org.flywaydb.flyway' version '5.0.7'
-  id 'org.springframework.boot' version '2.0.2.RELEASE'
+  id 'org.springframework.boot' version '1.5.10.RELEASE'
```

Comment:
This PR looks good to me just not sure if ff4j web console works.I had to use `1.5.4.RELEASE`  but didn't try `1.5.10 R` version

Reply:
Those are minor jumps, should be compatible. I believe in some projects there were several similar collisions with old hmcts library dependencies coming in and dragging various spring boot 1.5.* versions